src/components/DateRangePickerWrapper.tsx

```diff
@@ -35,6 +39,12 @@ const DateRangePickerWrapper: React.FunctionComponent<DateRangePickerWrapperProp
 
   const wrapperClasses = classNames(classes.dateRangePicker, wrapperClassName);
 
+  const { setActions, setDaysOfWeek, setMonths } = useDateRangeContext();
+
+  if (props.months) setMonths(props.months);
```

Comment:
Are we sure we want to trigger this on every render? Maybe we should wrap this into useEffect hook?